### DISCUSSION TITLE
A few questions about this project

### DISCUSSION THREAD

**Discussion by TheColorRed** [IanTerzo/Neutron#18] [General]
Hi, I found this project on reddit, and I had a few questions:

* Does this project support loading from a server such as: `http://localhost:8080`?
* Does this support bindings within the JavaScript itself without having to be inline? I would like to transfer images from OpenCV to to the JavaScript, and this is fairly slow when using websockets, so I was hoping for a way to do this faster without having to use the network to pass data between one another.

**Comment by IanTerzo** [comment]
Sorry for the late response,

1. Yes this project supports loading from localhost and remote servers. 
2. Binding within JavaScript should be possible both inline and not inline but I haven't looked into that so much.
